### PR TITLE
Cold-Coffee Recipe 

### PR DESCRIPTION
**Pull request by sakinarao**

### CODE REVIEW

File: src/recipes/cold-coffee.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        "Blend coffee powder and Sugar in a mixer jar with 2 teaspoons water until it turns light in colour.",
+        "Pour milk and Hersheys Syrup in the jar.",
+        "Blend the mixture until you get a good froth on the top.",
+        "Put the ice-cream in a serving glass.",
+        "Pour the coffee in the glass.",
+        "Garnish with the cocoa powder and some Hersheys Syrup."
+    ],
+    "image": "cold-coffee.jpg",
+    "keywords": [
+        "Cold Coffee",
```
Comment (alfg) [inline src/recipes/cold-coffee.json]:
```suggestion
        "cold Coffee",
```

File: src/recipes/cold-coffee.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        "Ice-cream"
```
Comment (alfg) [inline src/recipes/cold-coffee.json]:
```suggestion
        "ice-cream"
```

File: src/recipes/cold-coffee.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{
+    "name": "Cold-Coffee",
+    "description": "If you are a coffee lover you will relish it for sure.",
+    "github": "sakina_rao_",
```
Comment (alfg) [inline src/recipes/cold-coffee.json]:
```suggestion
    "github": "sakinarao",
```

**Comment by hamza962212** [comment]
This looks :sparkler:

**Comment by alfg** [comment]
Closing. Feel free to reopen with suggested changes.

**Pull request closed without merging**